### DISCUSSION TITLE
Error help

### DISCUSSION THREAD

**Discussion by Liam1236** [termux/termux-app#2709] [Q&A]
![Screenshot_20220412-013455](https://user-images.githubusercontent.com/102033487/162871468-3cdb8cc0-537a-4232-af1e-12f997ff8b34.png)
Sorry, can anyone help me?

**Comment by ChiefMikeK** [comment]
Need more info: 
___
since google has been restricting access more and more with each update since android 5  
 I would suggest moving your project to a termux home sub-directory 
- `~/npm/projects/lite`
- what os are you using? 
- model of your phone?
